Code/Framework/AzCore/AzCore/Memory/AllocatorInstance.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Module/Environment.h>
+#include <AzCore/Memory/IAllocator.h>
+#include <iostream>
```

Comment:
The iostream include is unneccessary

Reply:
Also the `IAllocator.h` include isn't needed either.

Only an IAllocator reference is between returned so that include can be omitted as well

Reply:
Fixed.